src/Domain/LogicalCoordinates.hpp

```diff
@@ -42,5 +45,5 @@ tnsr::I<DataVector, VolumeDim, Frame::Logical> logical_coordinates(
  */
 template <size_t VolumeDim>
 tnsr::I<DataVector, VolumeDim, Frame::Logical> interface_logical_coordinates(
```

Comment:
should this be renamed face_logical_coordinates?  this terminology may be a relic from the Bitbucket where an Interface was a class.  Should we reserve interface for internal faces, boundary_face for external faces, and face for a generic face?  @wthrowe @nilsdeppe thoughts?

Reply:
I like having three different, clearly defined terms to differentiate between the types of interfaces. Your suggestion sounds good to me :)

Reply:
We need better names for these things, and these suggestions are fine with me.  (The communication code is very inconsistent with all these terms so that isn't going to be very useful for inspiration here.)

Reply:
To refine the wording, how about _interface_, _boundary_ and _mortar_